includes/blocks/course-theme/class-course-theme.php

```diff
@@ -67,15 +68,13 @@ public function initialize_blocks() {
 		new Site_Logo();
 		new Notices();
 		new Focus_Mode();
-		if ( 'lesson' === get_post_type() ) {
-			new Prev_Lesson();
-			new Next_Lesson();
-			new Prev_Next_Lesson();
-			new Course_Progress_Counter();
-			new Course_Progress_Bar();
-			new Lesson_Actions();
-		} elseif ( 'quiz' === get_post_type() ) {
-			new Quiz_Back_To_Lesson();
-		}
+		new Course_Content();
+		new Prev_Lesson();
+		new Next_Lesson();
+		new Prev_Next_Lesson();
+		new Course_Progress_Counter();
+		new Course_Progress_Bar();
+		new Lesson_Actions();
+		new Quiz_Back_To_Lesson();
```

Comment:
Couldn't agree more! 💪

Reply:
We need all these blocks for quizzes too now, since completed quizzes use the normal `single.php` template. 
Also this area was a constant source of merge conflicts, and it's a very minor optimization.

Reply:
> We need all these blocks for quizzes too now, since completed quizzes use the normal course theme template

Ah, that's true! I forgot about that!